frontend/src/components/UsersList.js

```diff
@@ -1,25 +1,12 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
 import ListElement from "./listofusers/ListElement";
-import { fetchUsersWithApi } from "../communications/userApi";
 import { useTranslation } from "react-i18next";
+import useUsers from "./useUsers";
 
 const UsersList = () => {
-  const [users, setUsers] = useState([]);
+  const { users, fetchUsers } = useUsers();
```

Comment:
fetchUsers-t nem is használod :)

Reply:
Igen, gondoltam később szükség lehet rá, de törlöm, és majd visszaírom, ha ez bekövetkezne. :)

Reply:
Sose implementálunk olyasmit, amire nincs szükség. Majd ha kell, akkor elég megírni ;)